business/iter8.go

```diff
@@ -291,83 +365,24 @@ func (in *Iter8Service) GetAnalyticPort() int {
 	return analyticConfig.Port
 }
 
-func (in *Iter8Service) GetIter8RoutingReferences(namespace string, servicename string) (routingReference core_v1.ObjectReference, err error) {
-	promtimer := internalmetrics.GetGoFunctionMetric("business", "Iter8Service", "GetIter8Metrics")
-	defer promtimer.ObserveNow(&err)
-	istioCfg, err := in.businessLayer.IstioConfig.GetIstioConfigList(IstioConfigCriteria{
-		IncludeGateways:        true,
-		IncludeVirtualServices: true,
-		Namespace:              namespace,
-	})
-	rf := core_v1.ObjectReference{}
-	if err != nil {
-		return rf, err
+func validateVersion(requiredVersion string, installedVersion string) bool {
```

Comment:
I wonder if you want just to reuse existing function:
https://github.com/kiali/kiali/blob/master/status/versions.go#L80

And just make it public to be imported here.